interpreter/cling/lib/Interpreter/IncrementalParser.cpp

```diff
@@ -226,6 +227,12 @@ namespace cling {
     Preprocessor& PP = m_CI->getPreprocessor();
     DiagnosticsEngine& Diags = m_CI->getSema().getDiagnostics();
 
+    ASTReader* Reader = m_CI->getModuleManager().get();
+    ASTDeserializationListener* Listener = (ASTDeserializationListener*)m_Consumer;
```

Comment:
Please add an `assert(dynamic_cast< ASTDeserializationListener*>(m_Consumer))` to make sure this is really the actual type of `m_Consumer`.